src/create-router-for-api-method.ts

```diff
@@ -31,7 +34,12 @@ export const createRouterForApiMethod = (
       throw new Error(`Unknown method ${method} ${url}`);
     }
 
-    router[method as HttpMethod](url, getOperationHandler(methodConfig, dataAdapterStorage));
+    if(isCreateOperationWithFile(methodConfig)) {
+      const upload = multer({ dest: 'data/uploads/' })
```

Comment:
Get destination from configs. Also have a default config option too to configure from a single place...